draft-ietf-quic-transport.md

```diff
@@ -5495,6 +5497,16 @@ Reason Phrase:
   This SHOULD be a UTF-8 encoded string {{!RFC3629}}.
 
 
+## HANDSHAKE_DONE frame {#handshake-done-frame}
+
+The server uses the HANDSHAKE_DONE frame (type=0x1e) to signal confirmation of
+the handshake to the client.  The HANDSHAKE_DONE frame contains no additional
+fields.
+
+This frame can only be sent by the server. A server MUST treat receipt of a
```

Comment:
Can we add the following sentence of the start of this paragraph:

```
Servers MUST NOT send a HANDSHAKE_DONE frame before confirming
the handshake.
```

Reply:
Before **completing** the handshake.